.gitignore

```diff
@@ -1,5 +1,5 @@
 # MegaLinter
-report
+megalinter-reports
```

Comment:
I'm guessing these changes and the changes to `.pre-commit-config.yaml` were intended for a separate pull request?

Reply:
Yep.  Sorry.